### ISSUE TITLE
ddr gameboy noteskins?

### ISSUE THREAD

**Issue by arceuss** [MadkaT182/DDRNoteskins#2]
will there be any anytime soon

**Comment by MadkaT182** [comment]
Yes they are in plans, at this moment they just have the tap notes, are missing freeze and rolls, maybe in the next updates I will release them. Stay tuned!

**Comment by arceuss** [comment]
nice

**Comment by arceuss** [comment]
> Yes they are in plans, at this moment they just have the tap notes, are missing freeze and rolls, maybe in the next updates I will release them. Stay tuned!

https://www.spriters-resource.com/game_boy_gbc/dancedancerevolutiongbdisneymixjpn/sheet/155776/ are the tap notes from here by any chance btw
